routes/api/v1/forecast.js

```diff
@@ -0,0 +1,41 @@
+var _env = require('dotenv').config();
+var express = require('express');
+var router = express.Router();
+const fetch = require('node-fetch');
+
+const environment = process.env.NODE_ENV || 'development';
+const configuration = require('../../../knexfile')[environment];
+const database = require('knex')(configuration);
+
+router.get('/', (request, response) =>  {
+  database('users').first()
+    .then(user => {
+      let userCredential = request.body.api_key
+
+      let location = request.query.location
+      let geocode = process.env.GEOCODE_API_KEY
+
+      if (userCredential === user.apiKey) {
```

Comment:
This is different from what I did but I like it - you're code is clean and easily readable!

Reply:
Thank you, glad you like it.
I initially had set the user to a `const` outside of the router function. I learnt that Knex is promise-based therefore, I had to move the query inside the call.